transaction-view/src/instructions_frame.rs

```diff
@@ -57,17 +69,27 @@ impl InstructionsFrame {
             // Read the number of account indexes, and then update the offset
             // to skip over the account indexes.
             let num_accounts = optimized_read_compressed_u16(bytes, offset)?;
+            let accounts_offset = *offset as u16;
```

Comment:
Isn't it trivial to compute the offset for a compressed u16 if we know the decompressed value?

Reply:
🤔  good call - this probably is going to be as simple as something like:

```rust
if x > 127 { 2 } else { 1 }
```